mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -100,6 +94,11 @@ const GraphSidebar = (props: {
     */
     const [graphUpdatedNumber, setGraphUpdatedNumber] = useState(0)
 
+    useEffect(() => {
+        fscreen.addEventListener('fullscreenchange', () => {setGraphUpdatedNumber(old => old + 1)});
+        return () => fscreen.removeEventListener('fullscreenchange', () => setGraphUpdatedNumber(old => old + 1))
```

Comment:
I think you need the cleanup function, but not the callback to the cleanup function, right?